src/docs/samples/defining-parameters/groovy/gradle/plugins/build-parameters/build.gradle

```diff
@@ -24,7 +24,7 @@ buildParameters {
 buildParameters {
     bool('mybool') {
         description = 'Optional description of the bool parameter'
-        defaultValue= true // optional
+        defaultValue = false // optional
```

Comment:
Changed this to `false`, because I think that's the most common use case. Then you can switch it around just by using `-Pmybool` now.